docs/ref/settings.txt

```diff
@@ -2524,6 +2524,8 @@ A boolean that specifies whether to use the X-Forwarded-Host header in
 preference to the Host header. This should only be enabled if a proxy
 which sets this header is in use.
 
+.. note: This takes priority over USE_X_FORWARDED_PORT.
```

Comment:
Please use
```
:setting:`USE_X_FORWARDED_PORT`
```
to create a link for the setting. You're missing a second colon for `note::` so the block doesn't actually appear in the rendered version.